inc/databaseinstanceinjection.class.php

```diff
@@ -65,10 +67,35 @@ public function getOptions($primary_type = '')
 
         $tab                 = Search::getOptions(get_parent_class($this));
 
+        $tab[251]['name']          = __('Itemtype');
+        $tab[251]['field']         = 'itemtype';
+        $tab[251]['table']         = DatabaseInstance::getTable();
+        $tab[251]['linkfield']     = "itemtype";
+        $tab[251]['injectable']    = true;
+        $tab[251]['displaytype']   = 'text';
+        $tab[251]['checktype']     = 'text';
+
+        $tab[252]['name']          = __('Item');
+        $tab[252]['field']         = 'name';
+        $tab[252]['table']         = DatabaseInstance::getTable();
+        $tab[252]['linkfield']     = "items_name";
+        $tab[252]['injectable']    = true;
+        $tab[252]['displaytype']   = 'text';
+        $tab[252]['checktype']     = 'text';
+
+        $tab[253]['name']          = __('Path');
+        $tab[253]['field']         = 'path';
+        $tab[253]['table']         = DatabaseInstance::getTable();
+        $tab[253]['linkfield']     = "path";
+        $tab[253]['injectable']    = true;
+        $tab[253]['displaytype']   = 'text';
+        $tab[253]['checktype']     = 'text';
```

Comment:
This should be done from GLPI Core

You can take inspiration from the class `Agent.php` for the search option `itemtype` and `items_id`